### PR TITLE
Added functionality for Mark Messages as Read

### LINKED ISSUE #29: Mark Messages as Read
When you receive an incoming message from Webhooks, you can use the /messages endpoint to mark the message as read by changing its status to read. We need to add support for this in the library.

See https://developers.facebook.com/docs/whatsapp/cloud-api/guides/mark-message-as-read

### PR DESCRIPTION
**Pull request by pratsonii**
Fixes #29

### CODE REVIEW

File: src/test/java/com/whatsapp/api/impl/WhatsappBusinessCloudApiTest.java [hunk 1]
```diff
@@ -1123,6 +1124,31 @@ void testLocationMessage() throws IOException, URISyntaxException, InterruptedEx
...
+
+        var response = whatsappBusinessCloudApi.markMessageAsRead(PHONE_NUMBER_ID, message);
+
+        RecordedRequest recordedRequest = mockWebServer.takeRequest();
+        Assertions.assertEquals("POST", recordedRequest.getMethod());
+        Assertions.assertEquals("/" + API_VERSION + "/" + PHONE_NUMBER_ID + "/messages", recordedRequest.getPath());
+
+        JSONAssert.assertEquals(expectedJson, recordedRequest.getBody().readUtf8(), JSONCompareMode.STRICT);
+
+        Assertions.assertEquals(true, response.success());
```
Comment (Bindambc) [inline src/test/java/com/whatsapp/api/impl/WhatsappBusinessCloudApiTest.java:1149]: `Assertions.assertEquals(true, response.success())` can be simplified to `assertTrue()`

File: src/main/java/com/whatsapp/api/impl/WhatsappBusinessCloudApi.java [hunk 2]
```diff
@@ -113,5 +114,17 @@ public Response deleteMedia(String mediaId) {
         return executeSync(whatsappBusinessCloudApiService.deleteMedia(mediaId));
     }
 
+    /**
+     * Mark a message as Read
+     *
+     * @param phoneNumberId Represents a specific phone number.
+     * @param message       The {@link Message} object.
```
Comment (Bindambc) [inline src/main/java/com/whatsapp/api/impl/WhatsappBusinessCloudApi.java:121]: change javadoc text `{@link Message}` to `{@link ReadMessage}`, please

**Review by Bindambc — CHANGES_REQUESTED** [review]
Just some suggested changes. Thank you for the pull request.

**Review by Bindambc — APPROVED** [review]

**Comment by ferasawadi** [comment]
any ideas how to get the message id?

**Comment by Bindambc** [comment]
> any ideas how to get the message id?

Hello @ferasawadi, the messages ID can be obtained through a webhook. You will need to set up a webhook in advance to receive the messages and be able to respond to them or mark them as read.

You can find more detailed instructions on how to configure the webhook at the following links: [link1](https://developers.facebook.com/docs/whatsapp/business-management-api/guides/set-up-webhooks) and [link2](https://developers.facebook.com/docs/whatsapp/cloud-api/webhooks/payload-examples).

This library provides support for deserializing the payloads received through the webhook. You can find an example of this here: [link3](https://github.com/Bindambc/whatsapp-business-java-api/blob/main/src/test/java/com/whatsapp/api/examples/WebHookExample.java).

I hope this can help you. Please, feel free to open an issue if you have any further questions.

**Pull request merged**